package.json

```diff
@@ -98,12 +101,15 @@
     "@types/semver": "^7.3.4",
     "@types/sinon-chai": "^3.2.4",
     "@types/which": "^1.3.2",
-    "@typescript-eslint/eslint-plugin": "^4.28.4",
-    "@typescript-eslint/parser": "^4.28.4",
+    "@typescript-eslint/eslint-plugin": "^4.33.0",
+    "@typescript-eslint/parser": "^4.33.0",
+    "aws-sdk": "^2.674.0",
```

Comment:
erm.. let me doublecheck these. Probably a merge stuffup.

Reply:
Looks like aws-sdk+boxed-node moved to build and axios+browserify just got removed. fixing.